### PR TITLE
GLFWMouseAndKeyHandler: fix #377 in windows every click registers as …

### LINKED ISSUE #377: All mouse buttons seem to be recognized as button1
Move the camera works with all mouse buttons but should only be possible with button1. Also, commands wich are assigned to button 2 and 3 do not work.

In my initial investigation I found line 101 in `GLFWMouseAndKeyHandler` to be strange as the pressed button is always set to 0 same goes for the drag event above. However, this might not be relevant as the `getMask` method does not access that value. But that is also the point were I stopped my investigation as my knowledge of AWT input mask is lacking. Maybe the error is there somewhere. I don't see any code adding anything single button 2 and 3 clicks to the mask. And going with the debugger there and pressing left or right mouse buttons results in the same mask (1024).

Also scenerygraphics/sciview#356 of sciviewe might be related.

### PR DESCRIPTION
**Pull request by moreApi**
…left click and no scrolling. There were a few values in the wrong places.

### CODE REVIEW

**Comment by moreApi** [comment]
button2 and button3 do not map to the correct buttons

**Comment by moreApi** [comment]
they do now. Also the test that never worked corretly works now like it should ... i hope

**Review by skalarproduktraum — CHANGES_REQUESTED** [review]
One small change/correction, otherwise it looks good to me 👍 Thanks, @moreApi!

File: src/main/kotlin/graphics/scenery/controls/GLFWMouseAndKeyHandler.kt [hunk 1]
```diff
@@ -172,7 +181,8 @@ open class GLFWMouseAndKeyHandler(var hub: Hub?) : MouseAndKeyHandlerBase(), Aut
                 0,
                 0,
                 0, false, 0,
-                (xoffset.toFloat()*scrollSpeedMultiplier).toInt(), 0)
+                (yoffset.toFloat()*-scrollSpeedMultiplier).toInt(),
```
[the remark this answers is not in the document]
Comment (moreApi) [reply src/main/kotlin/graphics/scenery/controls/GLFWMouseAndKeyHandler.kt:184]: nope, horizontal scrolling is not supported by mouseWheelEvent.Those two parameters are different units which are not really clear and no one seems to use it properly in scenery anyway. So I fill both with something sensible and developes down the line have to fiddle around with the rates anyway

**Review by skalarproduktraum — APPROVED** [review]

**Pull request merged**